config/fixedwing/gemini2/cpd/apps/sens/CMakeLists.txt

```diff
@@ -5,3 +5,5 @@ buildliner_add_table(
     NAME sens_config
     SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/tables/sens_config.c
 )
+
+buildliner_add_app_unit_test_src(${APP_NAME} SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/tables/sens_config.c)
```

Comment:
Fantastic!  I don't remember showing anybody this pattern.  Did you figure this out on your own?  Either way, this is perfect.  Using this pattern allows us to unit test with the actual table.  Great job!  I can't tell you enough how happy I am to see the correct patterns used.